src/uu/free/src/units.rs

```diff
@@ -0,0 +1,139 @@
+use clap::ArgMatches;
+use std::fmt::{self, Display, Formatter};
+
+#[derive(Debug, PartialEq)]
+pub(crate) enum UnitMultiplier {
```

Comment:
there isn't a crate for this already?

Reply:
like https://crates.io/crates/human_bytes

Reply:
Oh, thanks for the reminder.

Reply:
did you look at it ? :)
thanks

Reply:
hum I don't think this crate is quite the way it needs to be, because it only supports one of SI and IEC. I used [bytesize](https://crates.io/crates/bytesize)